vercel.json

```diff
@@ -1,4 +1,15 @@
 {
+  "headers": [
+    {
+      "headers": [
+        {
+          "key": "Content-Type",
+          "value": "text/plain; charset=utf-8"
+        }
+      ],
+      "source": "/(.*\\.py)?"
```

Comment:
Looks like now we have the reverse problem.

```console
$ curl -I https://install-python-poetry-org-8wq4gtcf3-python-poetry.vercel.app/install-poetry.py
HTTP/2 200 
date: Wed, 09 Feb 2022 00:14:59 GMT
content-type: text/plain; charset=utf-8
...

$ curl -I https://install-python-poetry-org-8wq4gtcf3-python-poetry.vercel.app/
HTTP/2 200 
date: Wed, 09 Feb 2022 00:15:10 GMT
content-type: application/octet-stream
...
```

Reply:
Might just be easier to have two separate entries for `/` and `/*.py`.

Reply:
Right. I've tried variety of valid javascript regex syntax without any success.

Reply:
I have a feeling vercel treats `/` as a special case. /shrug